vm_deploy.sh

```diff
@@ -8,13 +8,13 @@ echo "==== Begin crownapp deploy ===="
 sudo true
 
 # Install homebreww
-if ( which brew > /dev/null ) 
-then
-  echo -e "\n==== Brew installed ====\n"
-else 
-  echo -e "\n==== Installing brew ====\n"
-  NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-fi
+# if ( which brew > /dev/null ) 
```

Comment:
you can leave this in if it doesn't do anything if brew is already installed. If it's not installed, might be good to have this in here to satisfy the dependency for install gsed.

Reply:
I was using this temporarily to test the crownapp_local.sh. the vm deploy we renamed yesterday to crownapp_deploy. I brought this file back also from my other repo.